CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CanHazFunny;
+
+public class Jester
+{
+
+    private IJokeService JokeService { get; set; } // set in constructor
+    private IConsoleOutput ConsoleOutput { get; set; }
+
+
+
+    public Jester(IJokeService IjokeService, IConsoleOutput IconsoleOutput)
```

Comment:
```suggestion
    public Jester(IJokeService jokeService, IConsoleOutput consoleOutput)
```
While you are typing the parameters as their interface types, the values actually being passed will not be the interface, but rather an instance of a class that implements that interface. So I would suggest naming your input variables without the `i`